repo2docker/buildpacks/base.py

```diff
@@ -14,14 +14,14 @@
 from traitlets import Dict
 
 TEMPLATE = r"""
-FROM buildpack-deps:bionic
+FROM nvidia/cuda:10.0-cudnn7-runtime-ubuntu18.04
```

Comment:
Todo (low priority): Need to check if we can use cuda 10.1 as the base version. Check if flatland/minerl will have any issues with that change.